### PR TITLE
So anyways, started colorin' moths

### PR DESCRIPTION
**Pull request by Deek-Za**
## About The Pull Request

So all the moths being a tan color? Kinda silly. This let's them be _beautiful again_
Adjusts some filepaths to the proper file.
Not that I... totally redid the sprite colors before realizing this or anything. Ha.. ha...(god kill me)

Also restructures the moth code to be a bit more nice aesthetically. At least in my opinion.

## How This Contributes To The Skyrat Roleplay Experience

Let's our beautiful moth brethren color up and express themselves proper. Until they end up in some sort of freak pizza delivery accident or something. Woah. That'd be weird.

Edit: Oh and default color seems to be #DBC18A(?) if anyone wants to know that for whatever reason

## Changelog

:cl: Deek-Za
fix: Moffs have color again
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/moth.dm [hunk 1]
```diff
@@ -1,23 +1,8 @@
...
-		HAS_MARKINGS,
-		MUTCOLORS
-	)
-	inherent_traits = list(
-		TRAIT_ADVANCEDTOOLUSER,
-		TRAIT_CAN_STRIP,
-		TRAIT_CAN_USE_FLIGHT_POTION,
-	)
+	default_mutant_bodyparts = list("fluff" = "None","wings" = ACC_RANDOM,"moth_antennae" = ACC_RANDOM)
+	species_traits = list(LIPS,HAS_FLESH,NOEYESPRITES,HAS_BONE,HAS_MARKINGS,MUTCOLORS)
```
Comment (nevimer) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/moth.dm]: Your branch is not in sync.

Comment (Deek-Za) [reply modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/moth.dm]: Odd. Heck it. Let's gooooo!

Comment (Deek-Za) [reply modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/moth.dm]: Seems to be a thing, but doesn't seem to work the colors properly. Everything goin' tan. Which is what I'm tryin' ta fix.
I'll tweak as needed but things seem to be rockin' it on my end.
image

File: code/modules/surgery/bodyparts/species_parts/moth_bodyparts.dm [hunk 2]
```diff
@@ -1,45 +1,37 @@
 /obj/item/bodypart/head/moth
-	icon = 'icons/mob/species/moth/bodyparts.dmi'
-	icon_state = "moth_head"
-	icon_static = 'icons/mob/species/moth/bodyparts.dmi'
+	icon_greyscale = 'modular_skyrat/master_files/icons/mob/species/moth_parts_greyscale.dmi'
```
Comment (Zonespace27) [inline code/modules/surgery/bodyparts/species_parts/moth_bodyparts.dm]: add skyrat edit comments to all of this.

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/surgery/bodyparts/species_parts/moth_bodyparts.dm (hunk 2 above)
Comment (Deek-Za) [reply code/modules/surgery/bodyparts/species_parts/moth_bodyparts.dm]: Done. Honestly, I didn't even begin to think TG had moths. My bad

File: code/modules/surgery/bodyparts/species_parts/moth_bodyparts.dm [hunk 3]
```diff
@@ -1,4 +1,44 @@
...
+	uses_mutcolor = TRUE
+
+/obj/item/bodypart/r_leg/moth
+	icon_greyscale = 'modular_skyrat/master_files/icons/mob/species/moth_parts_greyscale.dmi'
+	icon_state = "moth_r_leg"
+	limb_id = SPECIES_MOTH
+	uses_mutcolor = TRUE
+//SKYRAT EDIT END
+
+/*/obj/item/bodypart/head/moth
```
Comment (Zonespace27) [inline code/modules/surgery/bodyparts/species_parts/moth_bodyparts.dm]: gotta mention this as a skyrat removal as well

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/surgery/bodyparts/species_parts/moth_bodyparts.dm [hunk 4]
```diff
@@ -1,3 +1,44 @@
+//SKYRAT EDIT START: New Moth code for moth code colorable body parts
```
Comment (Gandalf2k15) [inline code/modules/surgery/bodyparts/species_parts/moth_bodyparts.dm]: Put this in the modular folder where the other limbs are. Search _mutant_boryparts.dm

**Comment by Gandalf2k15** [comment]
yo yo. fix de conflicys

**Comment by Deek-Za** [comment]
> yo yo. fix de conflicys

Away from home on some business.  I'll jump on finishing this as soon as I'm back.

**Pull request closed without merging**